app/assets/stylesheets/application.css.scss

```diff
@@ -14,6 +14,80 @@
  *= require_self
  */
 
+ #gif-overlay {
+   background-color: rgba(10, 10, 10, 0.5);
+   background-size: cover;
+   position: absolute;
+   width:  100vw;
+   height: 100vh;
+ }
+
+ #gif-container {
+  width:  100vw;
+  height: 100vh;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  position: fixed;
+  background-image: url(http://i.imgur.com/WTrMmzJ.jpg);
+}
+
+#gif-body {
+}
+
+#huge-page-button {
+  color:white;
+  width:400px;
+  height:100px;
+}
+#large-page-button {
+  color:white;
+  width:300px;
+  height:75px;
+}
+
+#large-page-button-text{
+  border:0;
```

Comment:
Properties should be ordered background, border, font-size, font-style, padding-top, text-transform<br>Colon after property should be followed by one space